README.md

```diff
@@ -30,6 +30,18 @@
 - run: `npm install` to download all dependencies.
 - run: `npm start` to start your local development server.
 
+## Errors on Deploy
```

Comment:
Wow, I've not had to deal with heap since working with java jvms. Great solution. If this happens on all deployments then we should make this the default in the package.json

Reply:
It's happening to enough students that it _might_ require a default. I'm not opposed to ensuring enough memory to boot up the application regardless of hardware.